common/app/model/EmailNewsletters.scala

```diff
@@ -2,419 +2,417 @@ package model
 
 import controllers.ListIds
 
-case class EmailSubscriptions(subscriptions: List[EmailSubscription])
-case class EmailSubscription(
+case class EmailNewsletters(subscriptions: List[EmailNewsletter])
+case class EmailNewsletter(
   name: String,
   theme: String,
   teaser: String,
   description: String,
   frequency: String,
   listId: String,
-  subscribedTo: Boolean = false,
-  subheading: Option[String] = None,
-  tone: Option[String] = None,
-  imageFilename: Option[String] = None,
-  signupPage: Option[String] = None,
-  exampleUrl: Option[String] = None
+  subscribedTo: (Iterable[String]) => Boolean,
```

Comment:
Why `Iterable` rather than `Seq` or `List`? (This is my own curiosity rather than a code review suggestion!)